doc/en/user/source/security/urlchecks.rst

```diff
@@ -3,65 +3,172 @@
 URL Checks
 ==========
 
-The URL checks page controls the checks that are performed on user provided URLs that
+The :guilabel:`URL External Access Checks` page controls the checks that are performed on user provided URLs that
 GeoServer will use to access remote resources.
 
 Currently, the checks are performed on the following functionality:
 
-- WMS GetMap requests with remote SLD stylesheets (``sld``` parameter)
+- WMS GetMap requests with remote SLD stylesheets (``sld`` parameter)
 - Remote icons referenced by styles (access to icons in the data directory is always allowed)
 - WMS GetMap in feature portrayal mode (``REMOTE_OWS`` and ``REMOTE_OWS_TYPE`` parameters)
 - WPS remote inputs, either as GET or POST requests
 
-More functionality might go under this kind of scrutiny in the future, check this page
-for eventual updates.
+Check this page for any additional remote service access checks added in the future.
 
 Configuration of URL checks
 ---------------------------
 
-The "URL checks" page under the Security section allows configuration of the URL checks.
+Navigate to :menuselection:`Data > URL Checks` page to manage and configure URL Checks.
```

Comment:
This one at least has a sensible rendering, bold with little arrowheads.

Reply:
I always write my page navigation text that way :)